src/lib/swapper/swappers/LifiSwapper/getTradeQuote/getTradeQuote.ts

```diff
@@ -39,6 +39,8 @@ export async function getTradeQuote(
     const sellLifiChainKey = lifiChainMap.get(sellAsset.chainId)
     const buyLifiChainKey = lifiChainMap.get(buyAsset.chainId)
 
+    const defaultLifiSwapperSlippage = getDefaultSlippagePercentageForSwapper(SwapperName.LIFI)
```

Comment:
yeah its odd that the swapper has to ask something else what its default slippage is, it should be able to go `this.defaultSlippage`

Reply:
might need to be passed in from the class though